src/pygram.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options as Firefox_Options
+from selenium.webdriver.common.keys import Keys
+from dotenv import dotenv_values
+from time import sleep
+import selenium_utils as su
+from login import login
+import logging
+import uuid 
+from feed import get_feed
+
+config = dotenv_values(".env")
+
+class session:
+    def __init__(self):
+        logging.basicConfig(filename="log.log",encoding='utf-8', level=logging.DEBUG)
```

Comment:
use [getLogger](https://docs.python.org/3/library/logging.html#logging.getLogger)
the config is set by the what runs the script